src/igvfd/tests/test_types_file.py

```diff
@@ -379,6 +379,7 @@ def test_file_assay_titles(
     )
     res = testapp.get(alignment_file['@id'])
     assert set(res.json.get('assay_titles', [])) == {'STARR-seq'}
```

Comment:
Shouldn't assay_titles be returning the assay_term.term_name ?

Reply:
yes, but if you look at the fixture, term_names are STARR-seq and ATAC-seq